src/acl.c

```diff
@@ -1980,9 +1980,10 @@ sds ACLStringSetUser(user *u, sds username, sds *argv, int argc) {
     sds *acl_args = ACLMergeSelectorArguments(argv, argc, &merged_argc, &invalid_idx);
 
     if (!acl_args) {
-        return sdscatfmt(sdsempty(),
+        sds err = sdscatfmt(sdsempty(),
                          "Unmatched parenthesis in acl selector starting "
                          "at '%s'.", (char *) argv[invalid_idx]);
+        return sdsmapchars(err, "\r\n", "  ",  2);
```

Comment:
i don't like this solution (with no explanation what it's for...
i think it should be inside an addReplySdsSafe function.

i suppose we only have this concern when returning replies to clients, not when errors to a module API.

Reply:
ok, will look into that.

Reply:
think I did this added a addReplyErrorSdsSafe and use it in acl return to user.